### PR TITLE
Fixes the deprecated wallet list

### PR DESCRIPTION
**Pull request by sneakzttv**
As https://registry.walletconnect.com/data/wallets.json is now deprecated, I've replaced it with wallet connects new version and added our default project id which is needed for the url.

### CODE REVIEW

**Review by boorich — APPROVED** [review]
👍🏼

**Review by kantagara — APPROVED** [review]

File: src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs [hunk 1]
```diff
@@ -231,7 +231,7 @@ private void BuildWalletConnectConfig()
 
     private IEnumerator FetchSupportedWallets()
     {
-        using (UnityWebRequest webRequest = UnityWebRequest.Get("https://registry.walletconnect.com/data/wallets.json"))
+        using (UnityWebRequest webRequest = UnityWebRequest.Get("https://explorer-api.walletconnect.com/v3/wallets?projectId=f4bff60eb260841f46b1c77588cd8acb"))
```
Comment (robGG1997) [inline src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs:234]: Can this be replaced with `$"https://explorer-api.walletconnect.com/v3/wallets?projectId={projectId}"` makes it reference the serialized project Id

**Review by robGG1997 — CHANGES_REQUESTED** [review]

File: src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs (hunk 1 above)
Comment (sneakzttv) [reply src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs:234]: can do

Comment (sneakzttv) [reply src/UnitySampleProject/Assets/Samples/web3.unity SDK/2.5.0/Web3.Unity Samples/Scripts/Scenes/WalletConnectLoginProvider.cs:234]: Done

**Review by robGG1997 — APPROVED** [review]

**Pull request merged**